src/main/java/Duke/TaskManager.java

```diff
@@ -0,0 +1,103 @@
+package Duke;
+
+public class TaskManager {
+
+    private static final int RESIZE_FACTOR = 2;
+    private static int NUM_TASK;
```

Comment:
Please follow the naming conventions for regular variables. It should be pascalCase for this one since it is not a `final` variable